style.css

```diff
@@ -0,0 +1,459 @@
+html,
+body {
+  width: 100%;
+  height: 100%;
+}
+
+html {
+  font: 15px 'montserratregular';
+}
+
+body {
+  margin: 0;
+  position: relative;
+  min-width: 320px;
+  max-width: 1440px;
+  align-items: center;
+}
+
+h1,
+h2 {
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  font-family: 'montserratsemibold';
+  text-decoration: none;
+  color: black;
+}
+
+button {
+  border: none;
+  background: none;
+  cursor: pointer;
+  outline: none;
+}
+
+header {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;	
+}
+
+.header__pic {
+   margin: 10px;
+}
+
+.menu {
+  margin: 10px;
+  padding: 0;
+  display: flex;
+		
+}
+
+.header__aside {
+  display: flex;
+  margin: 30px 30px;
+}
+
+.header__menu-link {
+  margin: 20px;
+  padding: 0;
+  color: black;
+}
+
+.header__menu-link:hover {
+  color: #e51937;
+  text-decoration: underline;
+}
+
+#light {
+  color:  #e51937;
+}
+
+#light::after {
+  content: '_______';
+  color: #e51937;
+  display: flex;
+  flex-diretion: column;
+  justify-content: center;
+}
+
+.header__line {
+  position: absolute;
+  left: 458px;
+  top: 50px;
+}
+
+.menu__icon {
+  width: 24px;
+  height: 15px;
+  margin: 0 5px;
+}
+
+.main__title {
+  font-size: 90px;
+  font-family: 'montserratsemibold';
+  margin: 40px 0 0 0;
+  padding: 0;
+  text-align: center;
+}
+
+.main__subtitle {
+  font: 15px 'montserratmedium';
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+
+.main__pic {
+  width: 100%;
+  margin: -110px 0 0 0;
+	
+}
+
+.main__info {
+  display: flex;
+  justify-content: center;
+}
+
+.info__content {
+  display: flex;
+}
+
+.main__content {
+  margin: 30px 0;	
+}
+
+.main__content-item {
+  margin: 0 20px;
+  display: flex;	
+}
+
+.content {
+  display: flex;
+  align-items: center;
+  margin: 0 20px;
+}
+
+.content__item {
+  margin-right: 5px;
+  display: flex;	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)